src/foremast/utils/elb.py

```diff
@@ -15,17 +15,17 @@
 #   limitations under the License.
 """Search for ELB DNS name."""
 import logging
+from tryagain import retries
 
 import boto3
-from tryagain import retries
 
 from ..exceptions import SpinnakerElbNotFound
 from ..utils.gate import gate_request
 
 LOG = logging.getLogger(__name__)
 
 
-@retries(max_attempts=5, wait=2, exceptions=(AssertionError, SpinnakerElbNotFound))
+@retries(max_attempts=6, wait=10, exceptions=(AssertionError, SpinnakerElbNotFound))
```

Comment:
Lets leave max attempts the same, but increase the wait to 5 seconds.

Reply:
I have replaced this with the backoff library per my initial recommendation in my pull request. I have changed the max attempts back to the original values per request